dfc.providerportal.findanapprenticeship/src/dfc.providerportal.findanapprenticeship.unittests/Integration/GetApprenticeshipsAsProviderIntegrationTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Dfc.Providerportal.FindAnApprenticeship.Functions;
+using Dfc.Providerportal.FindAnApprenticeship.Helper;
+using Dfc.Providerportal.FindAnApprenticeship.Interfaces.Helper;
+using Dfc.Providerportal.FindAnApprenticeship.Interfaces.Services;
+using Dfc.Providerportal.FindAnApprenticeship.Models;
+using Dfc.Providerportal.FindAnApprenticeship.Services;
+using Dfc.Providerportal.FindAnApprenticeship.Settings;
+using FluentAssertions;
+using LazyCache;
+using Microsoft.ApplicationInsights;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Moq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Dfc.ProviderPortal.FindAnApprenticeship.UnitTests.Integration
+{
+    public class GetApprenticeshipsAsProviderIntegrationTests
+    {
+        private readonly TelemetryClient _telemetryClient;
+        private readonly IAppCache _appCache;
+        private readonly Mock<IConfiguration> _configuration;
+        private readonly Mock<ICosmosDbHelper> _cosmosDbHelper;
+        private readonly Mock<IOptions<CosmosDbCollectionSettings>> _cosmosSettings;
```

Comment:
```suggestion
        private readonly Mock<IOptions<CosmosDbCollectionSettings>> _cosmosSettings = new Mock<IOptions<CosmosDbCollectionSettings>>();
```
I think this style is tidier

Reply:
Do settings need to be mocked? Why not just `IOptions<CosmosDbCollectionSettings>` without the `Mock<T>` wrapper? (Using `Options.Create<T>()` to make an instance.)

Reply:
Good shout, I didn't think to do that.